magicctapipe/io/io.py

```diff
@@ -172,6 +260,91 @@ def get_stereo_events(
     return event_data_stereo
 
 
+
+def get_stereo_events(
```

Comment:
`get_stereo` is basically a subset of `get_stereo_events`... why not adding a flag to `get_stereo_events` so that if that flag is true, then multiplicity and combo types are added to the dataframe? If false, simply return the dataframe after the multiplicity query.

Reply:
Yes, I duplicated it months ago and I forgot to fix it, sorry!!